### PR TITLE
Add test coverage for `bip85.py` early return paths

### PR DESCRIPTION
**Pull request by bc1cindy**
Covers lines 48 and 119 in bip85.py with cancellation scenarios:
- Line 48: Cancel at mnemonic word count selection
- Line 119: Cancel at index input for base64 password

### Changes made to:
- [ ] Code
- [x] Tests
- [ ] Docs
- [ ] CHANGELOG

### Did you build the code and tested on device?
- [ ] Yes

### What is the purpose of this pull request?
- [ ] Bug fix
- [ ] New feature
- [ ] Docs update
- [ ] Other

### CODE REVIEW

File: tests/pages/home_pages/test_bip85.py [hunk 1]
```diff
@@ -144,6 +144,18 @@ def test_bip85_bip39_mnemonic_derivation(mocker, amigo, tdata):
             "a620a1bc",
             True,
         ),
+        # 8 - Line 48 coverage: Cancel at mnemonic word count selection
```
Comment (qlrd) [inline tests/pages/home_pages/test_bip85.py]:
```suggestion
        # 8 - Cancel at mnemonic word count selection
```

File: tests/pages/home_pages/test_bip85.py [hunk 2]
```diff
@@ -517,6 +530,18 @@ def test_bip85_base64_password_derivation(mocker, amigo, tdata):
             ],
             "dKLoepugzdVJvdL56ogNV\n\nIndex: 0\nLength: 21",
         ),
+        # 7 - Line 119 coverage: Cancel at index input
```
Comment (qlrd) [inline tests/pages/home_pages/test_bip85.py]:
```suggestion
        # 7 - Cancel at index input
```

File: tests/pages/home_pages/test_bip85.py [hunk 3]
```diff
@@ -144,6 +144,18 @@ def test_bip85_bip39_mnemonic_derivation(mocker, amigo, tdata):
             "a620a1bc",
             True,
         ),
+        # 8 - Line 48 coverage: Cancel at mnemonic word count selection
+        (
+            tdata.SINGLESIG_12_WORD_KEY,
+            [
+                BUTTON_ENTER,  # BIP39 Mnemonic
+                BUTTON_PAGE_PREV,  # Move to "< Back"
+                BUTTON_ENTER,  # Confirm "< Back" - triggers line 48
```
Comment (qlrd) [inline tests/pages/home_pages/test_bip85.py]:
```suggestion
                BUTTON_ENTER,  # Press "< Back"
```

File: tests/pages/home_pages/test_bip85.py [hunk 4]
```diff
@@ -517,6 +530,18 @@ def test_bip85_base64_password_derivation(mocker, amigo, tdata):
             ],
             "dKLoepugzdVJvdL56ogNV\n\nIndex: 0\nLength: 21",
         ),
+        # 7 - Line 119 coverage: Cancel at index input
+        (
+            tdata.SINGLESIG_12_WORD_KEY,
+            [
+                BUTTON_PAGE,  # Move to "Base64 Password"
+                BUTTON_ENTER,  # Select "Base64 Password"
```
Comment (qlrd) [inline tests/pages/home_pages/test_bip85.py]:
```suggestion
                BUTTON_ENTER,  # Press "Base64 Password"
```

**Comment by bc1cindy** [comment]
thanks! @qlrd 

done

File: tests/pages/home_pages/test_bip85.py [hunk 5]
```diff
@@ -144,6 +144,18 @@ def test_bip85_bip39_mnemonic_derivation(mocker, amigo, tdata):
...
+        # 8 - Cancel at mnemonic word count selection
+        (
+            tdata.SINGLESIG_12_WORD_KEY,
+            [
+                BUTTON_ENTER,  # BIP39 Mnemonic
+                BUTTON_PAGE_PREV,  # Move to "< Back"
+                BUTTON_ENTER,  # Press "< Back"
+            ],
+            None,  # No mnemonic generated
+            None,  # No fingerprint
```
Comment (qlrd) [inline tests/pages/home_pages/test_bip85.py]: If we do not generate a bip85 mnemonic, we still have a fingerprint from `tdata.SINGLESIG_12_WORD_KEY`:

```suggestion
            "a620a1bc",
```

**Review by qlrd — CHANGES_REQUESTED** [review]
> i'm using cases on line 194 because this line is outside the loop and specifically needs cases[1] (the second test case) for the hidden mnemonic test, not the last case from the loop which has None value. Required for test to pass.

truth, good catch. In this case we should add a line, before the 

```python
ctx = create_ctx(mocker, cases[1][1], wallet)
```

because `wallet` is possible unbounded (the wallet is defined in the loop, not outside the loop). In this case, those changes should be applied to other tests that show the same behaviour. 

As mentioned in the reviewed code, we can replace `None` for a `string` fingerprints since the original ones from keys will be displayed (and not the generated ones). 

Also, we have some unused constants in another tests that could be removed.

To accelerate your job, i made a little patch that could be applied directly, this covers the original fingerprints on unloaded bip85, the possible unbounded wallets in hidden mnemonic tests and the remotion of unused constants: 

```diff
diff --git a/tests/pages/home_pages/test_bip85.py b/tests/pages/home_pages/test_bip85.py
index 34df2860..a5788510 100644
--- a/tests/pages/home_pages/test_bip85.py
+++ b/tests/pages/home_pages/test_bip85.py
@@ -27,7 +27,7 @@ def test_bip85_bip39_mnemonic_derivation(mocker, amigo, tdata):
                 BUTTON_ENTER,  # Confirm
             ],
             None,
-            None,
+            "a620a1bc",
             False,
         ),
         # 1 - 12w, load
@@ -103,7 +103,7 @@ def test_bip85_bip39_mnemonic_derivation(mocker, amigo, tdata):
                 BUTTON_ENTER,  # Confirm
             ],
             None,
-            None,
+            "a620a1bc",
             False,
         ),
         # Assert if no index is assigned it stays in the loop
@@ -153,7 +153,7 @@ def test_bip85_bip39_mnemonic_derivation(mocker, amigo, tdata):
                 BUTTON_ENTER,  # Press "< Back"
             ],
             None,  # No mnemonic generated
-            None,  # No fingerprint
+            "a620a1bc",
             False,  # Not loaded
         ),
     ]
@@ -186,6 +186,7 @@ def test_bip85_bip39_mnemonic_derivation(mocker, amigo, tdata):

     # Test with hidden mnemonic setting enabled
     Settings().security.hide_mnemonic = True
+    wallet = Wallet(cases[1][0])
     ctx = create_ctx(mocker, cases[1][1], wallet)
     bip85_ui = Bip85(ctx)
     mocker.spy(bip85_ui, "display_mnemonic")
@@ -370,12 +371,14 @@ def test_bip85_bip39_mnemonic_derivation_m5(mocker, m5stickv, tdata):
     print("last case hide_mnemonic")
     # Test with hidden mnemonic setting enabled
     Settings().security.hide_mnemonic = True
+    wallet = Wallet(cases[1][0])
     ctx = create_ctx(mocker, cases[1][1], wallet)
     bip85_ui = Bip85(ctx)
     mocker.spy(bip85_ui, "display_mnemonic")
     bip85_ui.export()
     ctx.display.draw_centered_text.assert_called_with(
-        FINGERPRINT_SYMBOL + THIN_SPACE + "%s" % case[3], color=theme.highlight_color
+        FINGERPRINT_SYMBOL + THIN_SPACE + "%s" % cases[1][3],
+        color=theme.highlight_color,
     )


@@ -383,9 +386,7 @@ def test_bip85_base64_password_derivation(mocker, amigo, tdata):
     from krux.pages.home_pages.bip85 import Bip85
     from krux.wallet import Wallet
     from krux.input import BUTTON_ENTER, BUTTON_PAGE, BUTTON_PAGE_PREV
-    from krux.settings import TEST_TXT
     from embit import bip32
-    from embit.networks import NETWORKS

     cases = [
         # case
```

**Comment by bc1cindy** [comment]
thanks @qlrd 

done!
